### PR TITLE
Updated date nag to handle short date cases.

### PR DESCRIPTION
**Pull request by bharney0**
Fixed date nag for updating ms.date metadata to current date for cases where the date input is a short hand version of the date. For example: 2/18/2021 instead of 02/18/2021. This type case is handled for short month and day type dates.

AB#376941

### CODE REVIEW

**Comment by lamebrain** [comment]
do you mind adding a test for this?

**Comment by bharney0** [comment]
@lamebrain added a test for the short date case.

File: packages/docs-markdown/src/test/suite/helper/metadata.test.ts [hunk 1]
```diff
@@ -152,4 +152,31 @@ suite('Metadata Helper', () => {
...
+		await config.update('enableMetadataDateReminder', true);
+		const stubShowInformationMessage = sinon.stub(window, 'showInformationMessage');
+		stubShowInformationMessage.onCall(0).resolves(Promise.resolve(undefined) as any);
+		const filePath = resolve(
+			__dirname,
+			'../../../../../src/test/data/repo/articles/test/metadata5.md'
+		);
+		await loadDocumentAndGetItReady(filePath);
+		const clock = sinon.useFakeTimers({
+			now: new Date(2021, 1, 2, 0, 0)
```
Comment (lamebrain) [inline packages/docs-markdown/src/test/suite/helper/metadata.test.ts:175]: is this the right date?

Comment (bharney0) [reply packages/docs-markdown/src/test/suite/helper/metadata.test.ts:175]: Thats the date for 02/02/2021, The first parameter are as follows `new Date(year, month(zero based), day date, N/A, N/A)`

**Review by lamebrain — APPROVED** [review]

**Pull request merged**